### ISSUE TITLE
Feature request: start swank by dispatch 

### ISSUE THREAD

**Issue by antonvesnin** [kovisoft/slimv#12]
Hi. Thanks for the great plugin. Is it possible to add start swank server through dispatch.vim - https://github.com/tpope/vim-dispatch  if it is available? Or can I add it just by editing vimrc?

**Comment by ELLIOTTCABLE** [comment]
:+1:!

`vim-dispatch` is pretty popular; it wouldn't be a bad idea to add support for this natively. (i.e. `:Dispatch swank`, and information in the README to that effect.)

**Comment by ELLIOTTCABLE** [comment]
Sorry, it wouldn't be `:Dispatch`, I made the mistake of repeating what I read above. It'd be `:Start!`; and I suppose all that'd be useful is adding a note to the README. :P

**Comment by kovisoft** [comment]
It's no problem to add support for it if I know

(1) how to detect the presence of vim-dispatch;
(2) what exact command should I build which starts the swank server.

(1) may be easy, e.g. by checking if command `:Dispatch` is defined
But I don't know (2) what command to build.
Is using vim-dispatch like starting a command in a new screen/tmux/xterm session?
I mean I have a command that would start the swank server when typed in the shell.
When I want to run this from within vim, then I can execute one of these (or similar):

`:! screen -X eval "command"`
`:! tmux new-window -d 'command'`
`:! xterm -e command &`

What would be the vim-dispatch equivalent?

**Comment by ELLIOTTCABLE** [comment]
The entire point of `vim-dispatch` is basically to make all the things you're doing in your readme, unnecessary. It handles any forking, or opening of new windows, or whatever, that may be necessary (for almost all `vim-dispatch` users, this means “open a new tmux split,” because it has special first-class tmux integration). So, basically, to start a background task in a new ‘window’ (or tmux pane), you'd use:

    :Start! command

I'm sure @tpope might be able to better explain than I, if he's listening to his notifications. (=

**Comment by tpope** [comment]
Yes, depending on the environment, `:Start command` or `:Start! command` could invoke something similar to any of those 3, (the former being foregrounded and the latter backgrounded).  Check with `exists(':Start') == 2`.
